rules/c/rule.md

```diff
@@ -0,0 +1,21 @@
+---
+type: rule
+title: Do you know to put any meeting join info at the top of the body?
+uri: meeting-join-info-at-the-top
```

Comment:
Folder name is not the same as the URI

Reply:
done - renamed folder

(I think my Ctrl C caused this 🤦‍♂️ )